src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/Expected/GenerateDesignerFileExpected.cs

```diff
@@ -27,8 +27,8 @@ static Resource()
 		public partial class Animator
 		{
 			
-			// aapt resource value: 0
-			public const int slide_in_bottom = 0;
+			// aapt resource value: 0x7F030002
+			public const int slide_in_bottom = 2130903042;
```

Comment:
This might be from ancient history, but I was wondering why we don't just use the hex value in C#:
```csharp
public const int some_resource = 0x7F030002;
```
Then we wouldn't need the comment?

It seems to compile fine: https://dotnetfiddle.net/3JPGU3

It's probably not worth changing this, I was mainly just wondering about it.

Reply:
@jonathanpeppers its probably because this is what the CodeDom outputs.